_books/ion-1-1/src/binary/values/list.md

```diff
@@ -31,57 +40,110 @@ B6 61 01 61 02 61 03
-│  │  ┌────── Opcode 0xF9 indicates a variable-length string. A FlexUInt length follows.
+│  │  ┌────── Opcode 0xF8 indicates a variable-length string. A FlexUInt length follows.
 │  │  │  ┌─────── Length: FlexUInt 20
 │  │  │  │   v  a  r  i  a  b  l  e     l  e  n  g  t  h     l  i  s  t
-FB 2d F9 29 76 61 72 69 61 62 6c 65 20 6c 65 6e 67 74 68 20 6c 69 73 74
+FA 2d F8 29 76 61 72 69 61 62 6c 65 20 6c 65 6e 67 74 68 20 6c 69 73 74
       └─────────────────────────────┬─────────────────────────────────┘
                           Nested string element
 ```
 
-##### Encoding of `null.list`
-```
-┌──── Opcode 0xEB indicates a typed null; a byte follows specifying the type
-│  ┌─── Null type: list
-│  │
-EB 09
-```
-
-#### Delimited Encoding
+### Delimited Encoding
 
-Opcode `0xF1` begins a delimited list, while opcode `0xF0` closes the most recently opened delimited container
+Opcode `0xF0` begins a delimited list, while opcode `0xEF` closes the most recently opened delimited container
 that has not yet been closed.
 
 ##### Delimited encoding of an empty list (`[]`)
 ```
-┌──── Opcode 0xF1 indicates a delimited list
-│  ┌─── Opcode 0xF0 indicates the end of the most recently opened container
-F1 F0
+┌──── Opcode 0xF0 indicates a delimited list
+│  ┌─── Opcode 0xEF indicates the end of the most recently opened container
+F0 EF
 ```
 
 ##### Delimited encoding of `[1, 2, 3]`
 ```
-┌──── Opcode 0xF1 indicates a delimited list
-│                    ┌─── Opcode 0xF0 indicates the end of
+┌──── Opcode 0xF0 indicates a delimited list
+│                    ┌─── Opcode 0xEF indicates the end of
 │                    │    the most recently opened container
-F1 61 01 61 02 61 03 F0
+F0 61 01 61 02 61 03 EF
    └─┬─┘ └─┬─┘ └─┬─┘
      1     2     3
 ```
 
 ##### Delimited encoding of `[1, [2], 3]`
 ```
-┌──── Opcode 0xF1 indicates a delimited list
-│        ┌─── Opcode 0xF1 begins a nested delimited list
-│        │        ┌─── Opcode 0xF0 closes the most recently
+┌──── Opcode 0xF0 indicates a delimited list
+│        ┌─── Opcode 0xF0 begins a nested delimited list
+│        │        ┌─── Opcode 0xEF closes the most recently
 │        │        │    opened delimited container: the nested list.
-│        │        │        ┌─── Opcode 0xF0 closes the most recently opened (and 
+│        │        │        ┌─── Opcode 0xEF closes the most recently opened (and 
 │        │        │        │    still open) delimited container: the outer list.
 │        │        │        │
-F1 61 01 F1 61 02 F0 61 03 F0
+F0 61 01 F0 61 02 EF 61 03 EF
    └─┬─┘    └─┬─┘    └─┬─┘
      1        2        3
 ```
 
+### Tagless-Element Lists
+
+Opcode `0x5B` indicates a tagless-element list. This is a compact encoding for homogeneous collections where all elements have the same type.
+
+The opcode is followed by:
+1. One or more bytes describing the tagless type:
+   - If the byte is in `0x00`-`0x47`: only one byte (the opcode) is present; this is the macro address
```

Comment:
The fact that the type may be a macro shape should be called out explicitly before this.